example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -75,6 +77,10 @@ const s3data = [
   '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606_ndvi.jpg'
 ];
 
+function findExecutionForGranule(taskInput, params) {
```

Comment:
Maybe a bit pedantic, but would it make sense for this to be named something like `executionForGranuleExists`? Seeing `find` leads me to believe there will be an index returned.

Reply:
Yeah I think you're right. I changed it to `isExecutionForGranuleId` which makes more sense since it's returning a bool